website/server/middlewares/auth.js

```diff
@@ -4,6 +4,9 @@ import {
 import {
   model as User,
 } from '../models/user';
+import nconf from 'nconf';
+
+const COMMUNITY_MANAGER_EMAIL = nconf.get('MAINTENANCE_MODE');
```

Comment:
I assume you meant `nconf.get('EMAILS:COMMUNITY_MANAGER_EMAIL');`
Good catch. I fixed this.